.gitignore

```diff
@@ -1,6 +0,0 @@
-.bundle
-.idea
-schema.rb
-spec/tmp
-pkg
-Gemfile.lock
```

Comment:
Sincere apologies that I overlooked this PR, it looks great! 

The only change we can't take is this modification to the .gitignore file since each of those is in there for a reason. If you can remove any changes to the .gitignore I'm happy to merge this. Also feel free to list yourself as a contributor at the bottom of the project README.

Reply:
@thuss whoops, that was accidental, fixed now

(i'm enjoying that there is just about exactly a year between our messages, i won't expect a response until May 2024)